include/deal.II/lac/la_parallel_vector.h

```diff
@@ -513,6 +513,24 @@ namespace LinearAlgebra
        * @ref GlossGhostedVector "vectors with ghost elements"
        */
       bool has_ghost_elements() const;
+
+      /**
+       * This method simply copies the data in the locally owned range from
+       * another distributed vector @p vec into the calling vector. As opposed
+       * to operator= that also includes ghost entries, this operation ignores
+       * the ghost range. The only prerequisite is that the local range on the
+       * calling vector and the given vector @p vec are the same on all
+       * processors. It is explicitly allowed that the two vectors have
+       * different ghost elements that might or might not be related to each
+       * other.
+       *
+       * Since no data exchange is performed, make sure that neither @p vec nor
+       * the calling vector have pending communications in order to obtain
+       * correct results.
+       */
+      template <typename Number2>
+      void copy_locally_owned_data(const Vector<Number2> &vec);
```

Comment:
It's not clear to me from the function name or the argument name whether you are copying *from* or *to* the given vector. I think that from the documentation you are calling from, but maybe call the function `copy_locally_owned_data_from` and name the argument `src` to make this clear?

Reply:
That's a good suggestion, thanks. I will do.